grails-app/conf/Config.groovy

```diff
@@ -128,7 +128,7 @@ openId {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    filterSource = 'database' // Todo - DN: This can go back to being a boolean when we've removed loading filters from the DB
```

Comment:
I think we're all on the same page. I'd like to leave this as-is for this PR but when we combine master and no_db this will be changed to a general configuration option and not treated as a feature toggle.